scene/resources/skeleton_profile.h

```diff
@@ -123,6 +126,10 @@ class SkeletonProfile : public Resource {
 
 	bool has_bone(const StringName &p_bone_name);
 
+#ifndef _3D_DISABLED
+	void initialize_from_skeleton(Skeleton3D *skeleton);
```

Comment:
```suggestion
	void initialize_from_skeleton(Skeleton3D *p_skeleton);
```
Prefix arguments with `p_`